Formula/mikutter.rb

```diff
@@ -0,0 +1,378 @@
+class Mikutter < Formula
+  desc "Extensible Twitter client."
+  homepage "http://mikutter.hachune.net"
+  url "http://mikutter.hachune.net/bin/mikutter.3.4.6.tar.gz"
+  sha256 "143318ad39ec60c9cf83e206f1a940e9464f0fda19c61e5e13a6cf327d1db076"
+  head "git://toshia.dip.jp/mikutter.git", :branch => "develop"
+
+  depends_on "gtk+"
+  depends_on "terminal-notifier" => :recommended
+  depends_on :ruby => "2.1+"
```

Comment:
The RubyRequirement doesn't use the `+`, it should just be `"2.1"` and that means 2.1 or higher.